.vib/dokuwiki/goss/goss.yaml

```diff
@@ -0,0 +1,29 @@
+file:
```

Comment:
IIRC, in the internal testing pipeline, we are checking some DokuWiki auth files are created. Maybe they can be useful here.

Apart from this, I have been thinking about the value of goss tests we are using to check subcomponents packaged with the asset and not the application itself. Although they are low-maintenance, I don't recall many issues where the asset failed because PHP or Apache did. I'm not sure if they add any tangible value to our test suites.

Reply:
OK, I see your point. I'll take a look at the internal pipeline tests for DokuWiki.

Reply:
I've modified the Goss tests, WDTY about them now?

Reply:
I like them, I think they can be useful to detect any issues in the init config.